app/templates/public.hbs

```diff
@@ -26,11 +26,13 @@
     <div class="ui stackable grid">
       <div class="three wide column" id="public-event-content">
         {{#if device.isMobile}}
-          <button {{action 'toggleMenu'}} class="ui basic icon button mobile only">
-            <i class="content icon"></i>
-          </button>
-          {{#if isMenuOpen}}
-            {{public/side-menu class='toggle menu' event=model}}
+          {{#if (and (not-eq session.currentRouteName 'public.cfs.new-session') (not-eq session.currentRouteName 'public.cfs.new-speaker') (not-eq session.currentRouteName 'public.cfs.edit-speaker') (not-eq session.currentRouteName 'public.cfs.edit-session'))}}
```

Comment:
Rather than using such conditionals in the templates, please use a computed property from controllers.

Reply:
The same condition is used in side-menu.hbs file for visibility of menu .

Reply:
This is really brittle

Reply:
I suspected so, @maze-runnar make a computed property and use it for the above condition too

Reply:
i  have made the computed property and it's working fine . may i make changes here ?